package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@connectedcars/eslint-plugin-react-i18n",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "node rules/checks.test.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@connectedcars/react-i18n": "^2.2.0"
```

Comment:
should this not be a dev dependency?

Reply:
It's a direct dependency of this repo in the `checks` rule so I don't believe so.